Yoast/wordpress-seo
# 10267 add cornerstone toggle to sidebar

## Summary

This PR can be summarized in the following changelog entry:

* Added the Cornerstone toggle from Yoast Components into the Yoast Sidebar. 

## Relevant technical choices:

* Merge this together with #[668](https://github.com/Yoast/yoast-components/pull/668) in components. 

## Test instructions

This PR can be tested by following these steps:

* See test instructions of #[668](https://github.com/Yoast/yoast-components/pull/668) in components

## Quality assurance


Fixes #10267 
Requires #[668](https://github.com/Yoast/yoast-components/pull/668)
